poznaj/stories/tests/test_views.py

```diff
@@ -76,3 +79,45 @@ def test_update_story(self):
         self.assertEqual(database_story.title, 'new_title')
         self.assertEqual(database_story.description, 'new_description')
         self.assertEqual(str(database_story.duration), '1:00:00')
+
+    def test_filter_story(self):
```

Comment:
@krzysztofzuraw, masz rację z tym testowaniem GeoDjango. :) Ale z drugiej strony - o ile dobrze rozumiem, to są takie testy end-to-end (nie czaję, czy w tych testach pod spodem jest baza, czy te factories to zapewniają taką bazę in-memory na potrzeby testów 😕 ) - widzę, że po prostu w teście robisz strzał do API tak, jakby robił je klient. Stąd pytanie, czy w takim "integracyjnym" ("integralnym"?) teście nie można po prostu sprawdzić, czy wynik jest oczekiwany (np. posortowane stories)?